OpenSim/Actuators/Thelen2003Muscle.h

```diff
@@ -148,6 +148,10 @@ OpenSim_DECLARE_CONCRETE_OBJECT(Thelen2003Muscle, ActivationFiberLengthMuscle);
         "The model governing the fiber and tendon kinematics.");
 
     /**@}**/
+    //Parallelism Method Override
+    //Note: Forces cannot be flagged as parallel if they modify (recalculate) system controls
+    //during calcForces
```

Comment:
You only need to put this note in the doxygen documentation for `Force`, I think.